dd_robot:
 joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 200
 joint_front_left_torque_controller:
  type: effort_controllers/JointEffortController
  joint: joint_front_left
  pid: {p: 0.0, i: 0.00, d: 0.0}
 joint_front_right_torque_controller:
  type: effort_controllers/JointEffortController
  joint: joint_front_right
  pid: {p: 0.0, i: 0.00, d: 0.0}
 joint_back_left_torque_controller:
  type: effort_controllers/JointEffortController
  joint: joint_back_left
  pid: {p: 0.0, i: 0.00, d: 0.0}
 joint_back_right_torque_controller:
  type: effort_controllers/JointEffortController
  joint: joint_back_right
  pid: {p: 0.0, i: 0.00, d: 0.0}
